assets/terraform-modules/bare-metal/flatcar-linux/kubernetes/worker.tf

```diff
@@ -0,0 +1,28 @@
+module "worker" {
+  source = "../../../worker"
+
+  count  = length(var.worker_names)
+  count_index = count.index
+
+  cluster_dns_service_ip = module.bootkube.cluster_dns_service_ip
+  ssh_keys               = var.ssh_keys
+  cluster_domain_suffix  = var.cluster_domain_suffix
+  ca_cert                = module.bootkube.ca_cert
+  apiserver              = format("%s.%s", var.cluster_name, var.k8s_domain_name)
+  kubelet_labels         = var.labels
+
+  clc_snippets = concat(var.clc_snippets[var.worker_names[count.index]], [
+    <<EOF
+storage:
+  files:
+  - path: /etc/hostname
```

Comment:
Setting hostname seems like a generic feature to me. I think we could add it to `node` module as optional, disabled by default, then enable it for tinkerbell and bare-metal.

Something like: `set_standard_hostname` maybe?

Reply:
done.